src/status_im/ui/screens/events.cljs

```diff
@@ -291,8 +291,7 @@
            (assoc db :view-id :intro :navigation-stack (list :intro))
            (let [{:keys [address photo-path name]} (first (sort-by :last-sign-in > (vals accounts)))]
              (-> db
-                 (assoc :view-id :login
-                        :navigation-stack (list :login))
```

Comment:
is it possible to return back from recovery screen?

Reply:
@flexsurfer fixed in `status-im.ui.screens.navigation/navigate-to-clean` now.